package.json

```diff
@@ -43,7 +43,7 @@
     "throttleit": "^1.0.0",
     "unlimited": "^1.1.0",
     "webtorrent": "0.x",
-    "webtorrent-app": "feross/webtorrent-app",
+    "webtorrent-desktop": "git+ssh://git@github.com/feross/webtorrent-desktop.git",
```

Comment:
Yeah, it stopped working for `feross/webtorrent-app` AND `feross/webtorrent-desktop`, `git+ssh` worked locally but broke travis, now `feross/webtorrent-desktop` seems to be working again. shrug